packages/app-account/src/containers/SignIn.js

```diff
@@ -3,25 +3,25 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import { Grid, Row, Col } from 'react-bootstrap'
-import { chainDispatch } from 'interbit-ui-tools'
+import { chainDispatch, selectors } from 'interbit-ui-tools'
```

Comment:
This component can be removed. I don't think it's used anywhere and it was combined with CreateAccount as both sign in and creating an account happen on that page now.